tests/manual_tests/test_rigetti_job_execution.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 University of Stuttgart
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""test in-request execution for aws"""
+
+from qunicorn_core.static.enums.assembler_languages import AssemblerLanguage
+from qunicorn_core.static.enums.provider_name import ProviderName
+from tests import test_utils
+
+IS_ASYNCHRONOUS: bool = False
+RESULT_TOLERANCE: int = 100
+
+"""Tests the execution of rigetti. quilc and qvm need to be running in server mode for this test to work"""
+
+
+def test_rigetti_local_simulator_braket_job_results():
```

Comment:
Auch noch die anderen Tests hinzufügen, Provider Rigetti, mit allen sprachen
Und auch bei den anderen tests noch quil hinzufügen

Reply:
Zurzeit können wir nur von qasm2 zu Quil übersetzen, eine Übersetzung in eine andere Sprache ist nicht möglich. Gab da leider von Seiten Rigetti keine einfachen Optionen. Zweiter Satz an Tests ist somit nicht möglich.